Sources/TuistCache/Cache/FileDownloader.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+import RxSwift
+import TSCBasic
+
+enum FileDownloaderError: LocalizedError {
```

Comment:
This has to conform to `FatalError` to be consistent with how we are handling errors in the codebase.

Reply:
Done.